### ISSUE TITLE
Update for Linux Mint 20?

### ISSUE THREAD

**Issue by chafey** [compulab/i3m-linux-daemon#1]
Does this work with Linux Mint 20?  If not, can it be updated?

**Comment by palday** [comment]
Based on my experience, the answer is unfortunately no.

**Comment by palday** [comment]
But now I've figured out what needs to be done.

First, follow the instructions for [19.2](http://www.fit-pc.com/wiki/index.php?title=Airtop:_Installing_Linux_Mint_19.2):

```bash
wget https://fit-iot.com/files/download/airtop/sw/I3M/Linux/airtop-fpsvc_3.0-2.deb
sudo dpkg -i airtop-fpsvc_3.0-2.deb
```

Then fix [an issue in the library paths](http://www.fit-pc.com/wiki/index.php?title=Installing_I3M_Linux_Daemon_on_Airtop3) before rebooting:
```bash
lib_path="/usr/lib/x86_64-linux-gnu"
[ -e $lib_path/libsensors.so.4 ] || sudo ln -s $lib_path/libsensors.so.*.0.0 $lib_path/libsensors.so.4
```

Then everything works on LM20.

**Comment by palday** [comment]
And yet another update: there are [instructions for Linux Mint 20 from CompuLab](http://fit-pc.com/wiki/index.php?title=Airtop:_Installing_Linux_Mint_20), including a few other things that were hard to get right on my own (like the audio jack).

**Comment by chafey** [comment]
Yeah, thank you for finding this information!  I followed the official instructions, but my system hangs on bootup when trying to start gpu-thermald via rc.local.  I am running on Pop!OS 20.04 so there could be some compatibility issue of some type.  I commented out gpu-thermald in rc.local for now, and the front panel is working.

**Comment by dmqgit** [comment]
@palday  @chafey Did you already try to get the fpsvc running on newer release of Linux Mint (i.e. 22.1)? I followed the steps of the in the Wiki. After that the front panel freezes (buttons do not work). In strace I see that the program tries to poll all i2c devices but gets timeouts or input /  output errors. Any idea? Thanks for your support.
